### ISSUE TITLE
v1.0.0 release?

### ISSUE THREAD

**Issue by preston** [IMSGlobal/caliper-ruby-public#1]
Can v1.0.0 actually be released? It's tagged here but the current release version is 0.9.3.15 from May, 2015, which isn't even tagged.

**Comment by preston** [comment]
..or if there's not a current maintainer I'd be happy to help with this.

**Comment by arwhyte** [comment]
Where is 0.9.3.15 referenced?

**Comment by preston** [comment]
It's [the most recent released version](https://rubygems.org/gems/ims_caliper/versions). There hasn't been a release in over a year.

**Comment by arwhyte** [comment]
No gem was pushed when 1.0.0 was released in October 2015.  Due to IMS hesitating on whether or not they wanted to produce such artifacts (caliper-java -> sonatype, caliper-js to npm, etc.).  I will bring this issue up.  A 1.1.0 release is scheduled for later this year (work occurring in a private repo per IMS rules).

**Comment by preston** [comment]
What's the point of a public implementation is if there's no intent to keep up with releases and active development is kept secret? As is, this sounds like a surefire way of ending up with subtly different, incompatible forked implementations. If IMS is going to claim the reference implementation, I would expect IMS to accept the corresponding responsibilities. If not, I suspect that the broader community could take ownership.

Publishing release builds only takes a minute once a tag is cut, so I'm not following why the community would be left with an intentionally outdated copy.

Thanks for all your work on this! 😄

**Comment by arwhyte** [comment]
I agree with you entirely and have proposed that IMS draw the curtain back and open up both their currently closed specification and reference implementation repositories.  I consider the IMS *-public repos extremely problematic.  The *-public repos are not the "canonical" repos in which day-to-day work takes place.  They disrupt otherwise well-understood Git/Github workflows and promote confusion.  They undermine the social contributor/review model of software / specs licensed as open-source and leave one with the impression that project / repo activity is moribund if not dead. 

As for providing binaries / release builds of the reference implementations I'm +1 since they simplify dependency management (acknowledging of course that no claims are being made that the ref implies represent production-reliable code) but IMS has been slow to grasp the advantages of providing them to developers.  

On the plus side IMS is reconsidering their current approach (the open-source ecology is new to them) based in part on recommendations that I and others have made.

**Comment by preston** [comment]
Any updates on this? The current situation doesn't make sense. :-/

**Comment by arwhyte** [comment]
Nope.  There is an upcoming IMS Quarterly Meeting in Orem, UT (mid-August) where I hope IMS clarifies their position regarding the openness issues that I and others have raised.
